src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+    private int number;
+
+    public LottoNumber(int value) {
+        numberRangeCheck(value);
```

Comment:
```suggestion
        numberRangeCheck(value);
```

이미 숫자 범위를 검증하고 있는 로직이 있습니다.
`Parser.numberParsing`가 필요한지 의문입니다 🤔 

```
public class LottoNumber { 
    private PositiveNumber number;
...
```
음수를 가지지 않게 하려면 양수 객체를 만들것 같습니다.